.stylelintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: 'stylelint-config-standard-scss',
```

Comment:
Here's the [config for `stylelint-config-standard-scss`](https://github.com/stylelint-scss/stylelint-config-standard-scss/blob/main/index.js). There are also [other rules](https://stylelint.io/user-guide/rules/) available plus [plugins](https://github.com/stylelint/awesome-stylelint#plugins) that we could consider enabling.

Reply:
[Q] Any rules that are missing that we should have turned on?

Reply:
I think some of the rules for "[Max and min](https://stylelint.io/user-guide/rules#max--min)"  would be useful for us as most of our current problems originate from that. 

[max-nesting-depth](https://stylelint.io/user-guide/rules/max-nesting-depth/)
[selector-max-attribute](https://stylelint.io/user-guide/rules/selector-max-attribute/)
[selector-max-class](https://stylelint.io/user-guide/rules/selector-max-class/)
[selector-max-combinator](https://stylelint.io/user-guide/rules/selector-max-combinators/)
[selector-max-type](https://stylelint.io/user-guide/rules/selector-max-type/) <- wondering if setting this to 0 would work
[selector-max-specificity](https://stylelint.io/user-guide/rules/selector-max-specificity/)

Reply:
Added max and min rules!

```
    'max-nesting-depth': 2,
    'selector-max-attribute': 2,
    'selector-max-class': 2,
    'selector-max-combinators': 3,
    'selector-max-type': 2,
```

Had multiple accounts of `selector-max-class` in `table` so added a disable comment and plan to fix them in a followup! 

I think having a max of two for each of these is a good thing to aim for in the future but we could discuss numbers for each in the future!

Reply:
`'selector-max-type': 2,` it's fine for now, but since we just changed some type selectors to classes, I feel like 1 would be more appropriate (if 0 does not work).